.github/workflows/ci.yaml

```
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        java-version: [ 8.0.292+10, 11.0.11+9 ]
+        java-version: [ 8.0.292+10, 11.0.11+9, 17-ea ]
```

Looks like Temurin only offers slightly more recent java 8 & 11 builds:
```suggestion
        java-version: [ 8.0.302+8, 11.0.12+7, 17-ea ]
```

---

When using a fixed version it shouldn't be the latest. As good practice customers or vendors using a open source library depend on a major release (fixed version) such as 8.0.192, and to get the latest you would just specify 8.
java-version: [ 8.0.192, 8, ... ]. if the build or test fails because of 8 (latest and greatest) you know it wasn't your code that broke. I hope that makes sense. Temurin doesn't support many major fixed versions prior to Sept2021 timeframe. Newly announced from Oracle is LTS is every 2 years and not 3 like previously known. So you will need to find out whether the Eclipse Foundation will follow the new cadence and builds they would maintain over time.
GH action setup-java version 1 was Azul zulu by default and has all versions available if you ever decide to make the switch. I assume Temurin will support them, but you might want to research just to make sure what to expect (their available versions). If at some point customers choose Zulu or Temurin and in need of support on production systems Azul supports both. 
https://www.azul.com/support-for-temurin/